docs/gettingstarted.rst

```diff
@@ -107,6 +107,13 @@ Quickstart
    Depending on what you're working on, you might want to run this weekly or
    maybe even daily.
 
+   .. Note::
+
+      This runs ftpscraper which takes 10 minutes to run. To save your time
```

Comment:
I think the formatting might not be working: https://github.com/mozilla-services/socorro/blob/bb66fd03c1ee576e5587cdfe8fb6794f354beddc/docs/gettingstarted.rst? Should the note text be within a gray box? Not sure if it's because there's 3 spaces instead of 2.

Reply:
Yes! So, "note" is not in the reST standard--it's defined in Sphinx:

http://www.sphinx-doc.org/en/stable/markup/para.html

GitHub is rendering reST using just reST and not the additional Sphinx stuff. So it just renders as a block with a title on GitHub.

You can see an example Note as rendered in our docs on RTD here:

https://socorro.readthedocs.io/en/latest/gettingstarted.html#updating-release-data